content/efcore/getting-started/samples/loading-related-data-tutorial/include-and-then-include-method-chain/Program.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
```

Comment:
Just `using System.Collections.Generic;`, sorry
Whenever I have unnecessary, it is just on the line that it is commented on